### ISSUE TITLE
feat: database initialization in local env

### ISSUE THREAD

**Issue by drodil** [drodil/backstage-plugin-qeta#13]
To ease the development, there should be a way to populate the local database with questions and answers.

**Comment by leonarhv** [comment]
@drodil Were you thinking about a pre-populated database for development? Or some route/button ta generates dummy data?

**Comment by drodil** [comment]
Yes exactly something like that. It always takes time to get to work with this plugin especially with memory only database, so this kind of feature would be very helpful.

**Comment by drodil** [comment]
There's now testdata.sql in the root but it must be manually run and doesn't contain that much data yet. But it's a start. Maybe it would be nice to have a script to run it in from the command line..

**Comment by drodil** [comment]
There is now a tool for this in `tools` directory
